Revert-tvOS/Sources/HomeCollectionViewController.swift

```diff
@@ -3,10 +3,24 @@
 
 import UIKit
  
-final class HomeCollectionViewController: UICollectionViewController {
+final class HomeCollectionViewController: UICollectionViewController, SettableHomeCollectionGroup {
+  var collectionGroup: String? {
+    didSet {
+      self.dataSource.filterGroups({
+        if let groupTitle = $0.title, collectionTitle = self.collectionGroup {
+          return groupTitle.localizedStandardContainsString(collectionTitle)
+        } else {
+          return false
+        }
+      })
+
+      self.collectionView?.reloadData()
+    }
+  }
+
   required init?(coder aDecoder: NSCoder) {
     self.dataSource = CollectionDataSource(
-      collection: self.collection,
+      collection: CollectableCollection<HomeItem>(items: .Home),
```

Comment:
`CollectableCollection`s are a specific kind of `Collection`s, one of the basic structures behind the data sources used throughout the app. @cuva might be able to give a better insight on this, but I guess they have been named like that from the beginning of the project.